packages/eas-cli/src/prompts.ts

```diff
@@ -9,6 +9,9 @@ export async function promptAsync<T extends string = string>(
   questions: Question<T> | Question<T>[],
   options: Options = {}
 ): Promise<Answers<T>> {
+  if (process.stdin.readableFlowing === null) {
+    throw new Error('Input is required, but stdin is not readable.');
```

Comment:
Maybe we should just use something like "Input is required, but command is in non-interactive mode"? Mentioning non-interactive could be more understandable compared to `stdin is not readable`?

Reply:
non-interactive is not exactly the same as the case where stdin is not readable